app/views/components/_label.html.erb

```diff
@@ -0,0 +1,40 @@
+<%
+  classes ||= ''
+  text_classes ||= ''
+  hint_text_classes ||= ''
+  html_for ||= ''
```

Comment:
Is `for` reserved?

Reply:
Mirroring the DOM here https://developer.mozilla.org/en-US/docs/Web/API/HTMLLabelElement/htmlFor